app/experimenter/experiments/tests/test_models.py

```diff
@@ -832,6 +832,21 @@ def test_is_ready_to_launch_true_when_reviews_and_sections_complete(self):
         )
         self.assertTrue(experiment.is_ready_to_launch)
 
+    def test_is_ready_to_launch_true_when_reviews_include_vp_review(self):
+        experiment = ExperimentFactory.create_with_status(
+            Experiment.STATUS_REVIEW,
+            review_science=True,
+            review_engineering=True,
+            review_qa_requested=True,
+            review_intent_to_ship=True,
+            review_bugzilla=True,
+            review_qa=True,
+            review_relman=True,
+            review_vp=True,
+            risk_partner_related=True,
+        )
+        self.assertTrue(experiment.is_ready_to_launch)
```

Comment:
Maybe we should have a test that `is_ready_to_launch` is `False` when `risk_partner_related` is `True` but `review_vp` is `False`?

Reply:
will do

Reply:
Yes in general both those tests are good ideas, we don't want to recreate those tests for every single pair of risk/review, so I think testing it once here with both   

    test_ready_to_launch_false_without_conditional_review
    test_ready_to_launch_true_with_conditional_review

should cover things well.